### ISSUE TITLE
Yolov10 : deacivating verbose on predict() ?

### ISSUE THREAD

**Issue by Morvanpa** [ultralytics/ultralytics#18681]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Looking to disable the prints from Yolo predictions :
`0: 288x512 1 person, 1 suitcase, 66.2ms
Speed: 2.5ms preprocess, 66.2ms inference, 0.3ms postprocess per image at shape (1, 3, 288, `512)`

I didn't find anything for Yolov10, tried solutions for v8 like : 
`import os`
`os.environ['YOLO_VERBOSE'] = 'False`
and
`model.predict(...,verbose=False)`

but neither worked, and model.predict doesn't have a verbose option anymore.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Morvanpa, thank you for your interest in Ultralytics 🚀! We recommend checking out our [Docs](https://docs.ultralytics.com) where you can find a wealth of information, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many common questions, such as disabling logging behavior, may already be answered there.

If this is a 🐛 Bug Report, kindly provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) along with the issue, as it helps us debug and resolve the problem more efficiently.

If your question relates to custom behavior or logging ❓, please share as much detail as possible, including the code snippet you're trying, any error messages, and your system setup. This will allow us to better understand your use case and assist effectively.

For real-time assistance, feel free to join our active [Discord](https://discord.com/invite/ultralytics) community 🎧. If you prefer detailed discussions, our [Discourse](https://community.ultralytics.com) forum is a great place to engage with other users. Additionally, you can explore and share ideas on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

To ensure you're using the latest features and fixes, upgrade to the latest `ultralytics` package along with its [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml). All updates work in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run seamlessly in the following up-to-date, verified environments (preconfigured with [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/)):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM ([GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMI ([AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Image** ([Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, our [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing! These tests verify correct behavior of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response to help guide you in the right direction 😊. An Ultralytics engineer will review your question and provide further assistance soon!

**Comment by Y-T-G** [comment]
`model.predict(...,verbose=False)` should work. What's your Ultralytics version?

**Comment by Morvanpa** [comment]
Well... 
I gave it some time and it now works ? 
Not sure what the issue was then, nor why it's resolved. `model.predict(...,verbose=False)` now works.

ultralytics version 8.4.48

**Comment by glenn-jocher** [comment]
@Morvanpa thank you for the update! Glad to hear it's working now. If you notice any further issues, feel free to reach out.

**Comment by kangxiao18** [comment]
```
    model = YOLOWorld(args.model_path)
    classnames = ["hazardous chemical transport vehicle", "hazchem truck", "dump truck","bus"]
    # classnames = ["car", "person"]
    model.set_classes(classnames)
    model.eval()results = model.predict(img, save=False, save_txt=False, conf=args.conf_thres, iou=args.iou_thres,verbose=False)
```
It's still print results like:

> 0: 384x640 1 bus, 16.5ms
> Speed: 1.5ms preprocess, 16.5ms inference, 0.8ms postprocess per image at shape (1, 3, 384, 640)
> 
> 0: 384x640 1 bus, 16.5ms
> Speed: 1.6ms preprocess, 16.5ms inference, 0.8ms postprocess per image at shape (1, 3, 384, 640)
> 
> 0: 384x640 (no detections), 16.5ms
> Speed: 1.5ms preprocess, 16.5ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)


Name: ultralytics
Version: 8.3.96

**Comment by Y-T-G** [comment]
@kangxiao18 I can't reproduce with latest version
